Gemfile.lock

```diff
@@ -92,10 +96,13 @@ GEM
       rexml
 
 PLATFORMS
+  arm64-darwin-23
```

Comment:
Is there a reason for ARM64 support?

Reply:
because I build it on my local machine for dev environment.  I guess that doesn't need to be in git but everytime I build it will be added.